packages/cli/README.md

```diff
@@ -15,31 +15,33 @@ This package is currently a work-in-progress and is set to `private`. Once it ma
-| `--namespaceResolver`     | `nsr`  | node request to a module that exports a stylable resolveNamespace function         | `@stylable/node` |
-| `--require`               | `r`    | require hook to execture before running                                            | `-`              |
-| `--optimize`              | `o`    | removes: empty nodes, stylable directives, comments                                | `false`          |
-| `--minify`                | `m`    | minify generated css                                                               | `false`          |
-| `--log`                   |        | verbose log                                                                        | `false`          |
-| `--diagnostics`           |        | print verbose diagnostics                                                          | `true`           |
+| Option                    | Alias  | Description                                                                            | Default Value                                |
+| ------------------------- | ------ | ----------------------------------------------------------------------------------     | ----------------                             |
+| `--version`               | `-v`   | show CLI version number                                                                | `boolean`                                    |
+| `--rootDir`               |        | root directory of project                                                              | `cwd`                                        |
+| `--srcDir`                |        | source directory relative to root                                                      | `./`                                         |
+| `--outDir`                |        | target directory relative to root                                                      | `./`                                         |
+| `--indexFile`             |        | filename of the generated index                                                        | `false`                                      |
+| `--cjs`                   |        | output commonjs modules (`.js`)                                                        | `true`                                       |
+| `--esm`                   |        | output esm modules (`.mjs`)                                                            | `false`                                      |
+| `--css`                   |        | output transpiled css files (`.css`)                                                   | `false`                                      |
+| `--stcss`                 |        | output stylable source files (`.st.css`)                                               | `false`                                      |
+| `--dts`                   |        | output definition files for the stylable source files (`.st.css.d.ts`)                 | `false`                                      |
```

Comment:
I think we need dts location option that is not related to the outDir

Reply:
I mean that we need to provide a way if someone wants to build the d.ts into dist vs src

Reply:
if we add the option, will it generate them only there and not in the output dir? we may want it to output to both source and target so that they are compatible in their content

Reply:
perhaps an option like, `srcDir | outDir | both` would work. I think we'll start without this, and see what real-world cases behave like.